pom.xml

```diff
@@ -605,7 +620,7 @@ under the License.
                   <compilerPlugin>
                     <groupId>org.scalameta</groupId>
                     <artifactId>semanticdb-scalac_${scala.version}</artifactId>
-                    <version>4.7.5</version>
+                    <version>4.8.8</version>
```

Comment:
Is this required for Spark 4.0?

Reply:
Yes, with Scala-2.13 and JDK 17, we need this version